server_admin/topics/clients/oidc/con-basic-settings.adoc

```diff
@@ -77,3 +75,19 @@ Domain URLs listed here are embedded within the access token sent to the client
 
 [[_back-channel-logout-url]]
 *Backchannel Logout URL*:: URL that will cause the client to log itself out when a logout request is sent to this realm (via end_session_endpoint). If omitted, no logout requests are sent to the client.
+
```

Comment:
Add this for the new UI

ifeval::[{project_community}==true]
*Backchannel logout revoke offline sessions*:: Specifies whether a `revoke_offline_access` event is included in the Logout Token when the Backchannel Logout URL is used. Keycloak will revoke offline sessions when receiving a Logout Token with this event.

endif::[]